android-o/AndroidPlayLocation/LocationUpdatesPendingIntent/LocationUpdatesPendingIntent/LocationUpdatesIntentService.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+using System;
+using System.Collections.Generic;
+using Android.App;
+using Android.Content;
+using Android.Gms.Location;
+using Android.Locations;
+using Android.OS;
+using Android.Util;
+
+namespace LocationUpdatesPendingIntent
+{
+	/**
+	 * Handles incoming location updates and displays a notification with the location data.
+	 *
+	 * For apps targeting API level 25 ("Nougat") or lower, location updates may be requested
+	 * using {@link android.app.PendingIntent#getService(Context, int, Intent, int)} or
+	 * {@link android.app.PendingIntent#getBroadcast(Context, int, Intent, int)}. For apps targeting
+	 * API level O, only {@code getBroadcast} should be used.
+	 *
+	 *  Note: Apps running on "O" devices (regardless of targetSdkVersion) may receive updates
+	 *  less frequently than the interval specified in the
+	 *  {@link com.google.android.gms.location.LocationRequest} when the app is no longer in the
+	 *  foreground.
+	 */
+	[Service(Label = "LocationUpdatesIntentService", Exported = false)]
+	[IntentFilter(new String[] { "LocationUpdatesPendingIntent.LocationUpdatesIntentService" })]
```

Comment:
I don't believe this IntentFilter is required. However, this sample is intended to demonstrate how to move away from using Android.App.PendingIntent.GetService, so any modification here is not really needed.

Reply:
Done